ui/narrative/methods/annotate_contigs/display.yaml

```diff
@@ -19,7 +19,7 @@ suggestions:
             []
     methods:
         related:
-            []
+            [annotate_contigset, annotate_domains_in_a_genome, annotate_plant_transcripts, reannotate_microbial_genome]
```

Comment:
@nlharris: no, they won't show up after the functionality of showing next/related apps is fixed. Because now methods/apps are identified by not only method id but also by module prefix (like "RAST_SDK/annotate_contigset" in case of https://github.com/kbaseapps/RAST_SDK/tree/master/ui/narrative/methods/annotate_contigset).